### ISSUE TITLE
Black Screen on SNES Super FX Games

### ISSUE THREAD

**Issue by rtomasa** [deltabeard/Haiyajan#22]
Games like StarFox using SFX chip do not work. DOOM is able to display the intro but then display black screen too.

**Comment by deltabeard** [comment]
Which libretro core are you using and what version?

**Comment by rtomasa** [comment]
Sorry I forgot to mention the tech details. It is snes9x_libretro.so running in a x64 bits PC running Fedora Linux

**Comment by deltabeard** [comment]
Thanks. I'll assume it's the version at https://github.com/libretro/snes9x/ with the latest commit of bc3bbb28c1723742d102c867e264574915d7c8ef.

It's been a while since I worked on this project, so I don't remember what state it's in with regards to libretro core compatibility. I remember having problems with a few libretro cores that worked in RetroArch but not with Haiyajan because they would call functions in an order that was not defined by the libretro specification. I also remember a problem whereby Haiyajan would not apply libretro settings, causing some libretro cores to not apply game specific workarounds, but I think I fixed that problem.

I'll have a look at this in the coming weeks. Thanks for your interest in this project.
